docs/visual-testing/integrations/java.md

```diff
@@ -0,0 +1,224 @@
+
+# Java WebDriver Integration
+
+## Introduction
+
+This guide requires an existing Java JUnit / TestNG setup.<br />
+You can alternatively take a look to our [example repository](#examples).
+
+Sauce Labs Visual provides an library allowing integration with [WebDriver](https://www.selenium.dev/documentation/webdriver/).
+
+Sauce Labs Visual plugin provides a library exposing a `VisualApi` object that provides actions:
+- `visual.sauceVisualCheck()`: Takes a screenshot and send it to Sauce Labs Visual for comparison.
+- `visual.sauceVisualResults()`: Waits for all diff calculations to complete and returns a summary of results.
+  See [Test results summary](#test-results-summary) for more details about summary format and sample usage.
+
+## Quickstart
+
+### Step 1: Add Sauce Labs Visual dependecy
+
+Add [Sauce Visual](https://central.sonatype.com/artifact/com.saucelabs.visual/java-client) dependency to your pom.xml
+  ```xml
+  <dependency>
+    <groupId>com.saucelabs.visual</groupId>
+    <artifactId>java-client</artifactId>
+    <version>0.3.121</version>
+    <scope>test</scope>
+  </dependency>
+  ```
+
+*Note: You can find the latest versions available [here](https://central.sonatype.com/search?q=com.saucelabs.visual).*
+
+### Step 2: Configure Visual Testing integration
+
+Declare a RemoteWebDriver and a VisualApi instance as class variables
+```java
+  private static VisualApi visual;
+  private static RemoteWebDriver driver;
+```
+
+Initialize `RemoteWebDriver` and `VisualApi` in `@BeforeAll` section, or `@BeforeSuite` if you're using TestNG
+
+<Tabs
+  defaultValue="JUnit"
+  values={[
+    {label: 'JUnit', value: 'JUnit'},
+    {label: 'TestNG', value: 'TestNG'},
+  ]}>
+  <TabItem value="JUnit">
+
+  ```java
+  @BeforeAll
+  public static void init() {
+      driver = new RemoteWebDriver(webDriverUrl, capabilities);
+      visual = new Builder(driver, sauceUsername, sauceAccessKey, DataCenter.US_WEST_1).build();
+  }
+  ```
+  </TabItem>
+  <TabItem value="TestNG">
+
+  ```java
+  @BeforeSuite
+  public static void init() {
+      driver = new RemoteWebDriver(webDriverUrl, capabilities);
+      visual = new Builder(driver, sauceUsername, sauceAccessKey, DataCenter.US_WEST_1).build();
+  }
+  ```
+  </TabItem>
+</Tabs>
+
+
+Don't forget to quit the WebDriver in `@AfterAll` section, or `@AfterSuite` if you're using TestNG 
+<Tabs
+  defaultValue="JUnit"
+  values={[
+    {label: 'JUnit', value: 'JUnit'},
+    {label: 'TestNG', value: 'TestNG'},
+  ]}>
+  <TabItem value="JUnit">
+
+  ```java
+  @AfterAll
+  public static void tearDown() {
+      if (driver != null) {
+          driver.quit();
+      }
+  }
+  ```
+  </TabItem>
+  <TabItem value="TestNG">
+
+  ```java
+  @AfterSuite
+  public static void tearDown() {
+      if (driver != null) {
+          driver.quit();
+      }
+  }
+  ```
+  </TabItem>
+</Tabs>
+
+
+### Step 3: Add visual tests in your tests
+
+Add a check to one of your tests:
+```java
+    @Test
```

Comment:
@kb-kerem 

People should also add the import, I'm not seeing that in the examples. If they now copy it it will not work